policyengine_us/tests/policy/baseline/gov/states/mt/tax/income/deductions/ctcc/mt_child_dependent_care_expense_deduction.yaml

```diff
@@ -0,0 +1,155 @@
+- name: Care expense greater than maximum amount with low AGI and 1 child
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 1
+    tax_unit_childcare_expenses: 2_401
+    mt_agi: 18_000 #No reduction needed
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 2_400
+
+- name: Care expense greater than maximum amount with moderate AGI and 1 child
+  period: 2021
+  input:
+    mt_child_dependent_care_expense_deduction_eligible_children: 1
+    tax_unit_childcare_expenses: 2_401
+    mt_agi: 20_000 #Partial reduction needed
+    state_code: MT
+  output:
+    mt_child_dependent_care_expense_deduction: 1_400
+  absolute_error_margin: 1
```

Comment:
move between period and input (others too)

and does it have to be so wide? would 0.01 work?